InAppMessagingDisplay/Example/Podfile

```diff
@@ -4,5 +4,6 @@ use_frameworks!
 target 'FiamDisplaySwiftExample' do
   platform :ios, '8.0'
   pod 'FirebaseInAppMessagingDisplay', :path => '../..'
+  pod 'FirebaseInAppMessaging', :path => '../..'
```

Comment:
Also need to depend on the FirebaseAnalyticsInterop from the local directory structure

Reply:
Thanks for the heads up.